src/neg_tests/thread_lin_tests_dsl.ml

```diff
@@ -0,0 +1,13 @@
+open Lin_tests_common_dsl
+
+(** This is a driver of the negative tests over the Thread module *)
+
+;;
+Util.set_ci_printing ()
+;;
+QCheck_runner.run_tests_main
+  (let count = 10000 in
```

Comment:
I think we should be extra careful when increasing the `count`.
In this case - even with a `count` of 10000, none of these tests still trigger a failure.
Looking at the CI logs in https://github.com/jmid/multicoretests/runs/7237062976?check_suite_focus=true
this takes `(516.8 +. 534.6 +. 390.7 +. 462.0)/. 60. = 31.735` minutes - that's half an hour extra!

Reply:
You're totally right, I was just trying to see whether a failure was finally triggered or if the problem is deeper.
The problem is definitely deeper...